mappings/net/minecraft/structure/MineshaftGenerator.mapping

```diff
@@ -93,6 +104,18 @@ CLASS net/minecraft/class_3353 net/minecraft/structure/MineshaftGenerator
 			ARG 4 maxX
 			ARG 5 y
 			ARG 6 z
+		METHOD method_33778 containsLiquid (Lnet/minecraft/class_1922;Lnet/minecraft/class_3341;)Z
+			ARG 1 world
+			ARG 2 box
+		METHOD method_33779 isAir (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338$class_2339;III)Z
+			COMMENT Determines whether the block at the given coordinates is air.
+			COMMENT
+			COMMENT <p>The passed position is mutated; the integer coordinates are used instead.
```

Comment:
It'd be better if you tell what mutation this method does

Reply:
Fixed in a449380f8b03ce6589a89cc7cb2ce161c837e302